ipfs-5-to-6/migration/config_conv.go

```diff
@@ -0,0 +1,154 @@
+package mg5
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"strings"
+)
+
+func convertFile(orig string, new string, convFunc func(ds map[string]interface{}) error) error {
+	in, err := os.Open(orig)
+	if err != nil {
+		return err
+	}
+	out, err := os.Create(new)
+	if err != nil {
+		return err
+	}
+	return convert(in, out, convFunc)
+}
+
+func convert(in io.Reader, out io.Writer, convFunc func(ds map[string]interface{}) error) error {
```

Comment:
@whyrusleeping I am confused, that is basically what I am doing.  Convert calls convFunc that does an inplace conversion on `map[string]interface`.  Maybe some better documentation to make it clear what is going on.